kim_cole/assignments/birthday_assignment/birthdayCountdown.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>Birthday Countdown</title>
+</head>
+Hello Kim Cole!
+<body>
+	<script type="text/javascript">
+
+	function birthdayCountdown(days){
+		var i;
+		for (var i=days; i >0; i--);
+			if (i >=30){
+				console.log(i  + days +' until my birthday. Such a long time... :(');
+			}
+			else 
+			if (i < 30 && i >=5) {
+				console.log(i + days + 'until my birthday. It\'s almost here!');
+			}
+			else 
```

Comment:
Thanks! here's my modifications:

<script type="text/javascript">
	"use strict";

	function birthdayCountdown(days){
		var i;
		for (var i=days; i >0; i--);
			if (i >=30){
				console.log(i  + days +' until my birthday. Such a long time... :(');
			}

			else if (i < 30 && i >=5) {
				console.log(i + days + 'until my birthday. It\'s almost here!');
			}
			else if(i < 5){
				console.log(i + days + ' DAYS UNTIL MY BIRTHDAY!!!');
			}

			else if(i=0){
			console.log('♪ღ♪*•.¸¸¸.•*¨¨*•.¸¸¸.•*•♪ღ♪¸.•*¨¨*•.¸¸¸.•*•♪ღ♪•*♪ღ♪░H░A░P░P░Y░ B░I░R░T░H░D░A░Y░░♪ღ♪*•♪ღ♪*•.¸¸¸.•*¨¨*•.¸¸¸.•*•♪¸.•*¨¨*•.¸¸¸.•*•♪ღ♪•«')
			}
	}
	

	// !test message for days until my birthday
	console.log(birthdayCountdown(60));
	console.log(birthdayCountdown(19));
	console.log(birthdayCountdown(2));
	console.log(birthdayCountdown(0));
	
	</script>

But it's still giving me the undefined on the next line. Here's a snapshot of my results on the console.
<img width="448" alt="screen shot 2017-03-05 at 6 11 23 pm" src="https://cloud.githubusercontent.com/assets/25779694/23593084/dfc27bd2-01cf-11e7-9849-82db5f7cbb7c.png">